src/flamenco/features/feature_map.json

```diff
@@ -159,7 +159,7 @@
   {"name":"vote_state_add_vote_latency","pubkey": "7axKe5BTYBDD87ftzWbk5DfzWMGyRvqmWTduuo22Yaqy"},
   {"name":"checked_arithmetic_in_fee_validation","pubkey": "5Pecy6ie6XGm22pc9d4P9W5c31BugcFBuy6hsP2zkETv","hardcoded":1},
   {"name":"last_restart_slot_sysvar","pubkey": "HooKD5NC9QNxk25QuzCssB8ecrEzGt6eXEPBUxWp1LaR"},
-  {"name":"reduce_stake_warmup_cooldown","pubkey": "GwtDQBghCTBgmX2cpEGNPxTEBUTQRaDMGTr5qychdGMj"},
+  {"name":"reduce_stake_warmup_cooldown","pubkey": "GwtDQBghCTBgmX2cpEGNPxTEBUTQRaDMGTr5qychdGMj","hardcoded":1},
```

Comment:
There is still one usage of this feature in [agave/sdk/src/feature_set.rs](https://github.com/anza-xyz/agave/blob/182d27f718e1d6df347bdbe60edaba6026f1c6b5/sdk/src/feature_set.rs#L1079-L1082)  - https://github.com/anza-xyz/agave/pull/470, which is used indirectly by the stake program.

I think in this case it is fine to treat this account as hardcoded, however, as the feature account will still exist in mainnet/testnet ledgers.